pages/room/[id].js

```diff
@@ -146,6 +157,24 @@ export default function Room(props) {
   useEffect(() => {
     if (roomSubscription.data) {
       const { roomUpdated } = roomSubscription.data
+      // check to see if a user was deleted
+      users.forEach((user1) => {
+        const foundUser = roomUpdated.users.findIndex(
+          (user2) => user2.id === user1.id
```

Comment:
Small consideration to rename user1 and user2 as it implies we are looking for 2 different users with the same id